data/sql/updates/pending_db_world/Roleplay.sql

```diff
@@ -0,0 +1,66 @@
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES 
+(@ENTRY, 0, 0, 1, 20, 0, 100, 0, 12227, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 'On player rewarded quest Doing Your Duty (12227) - Self: Talk Alright Ben. We\'ve retrieved the amberseeds again. You know ... (0) to invoker'),
+(@ENTRY, 0, 1, 0, 61, 0, 100, 0, 0, 0, 0, 0, 45, 0, 0, 0, 0, 0, 0, 10, 113907, 27071, 0, 0, 0, 0, 0, 'On player rewarded quest Doing Your Duty (12227) - Creature Benjamin Jacobs (27071) with guid 113907 (fetching): Set creature data #0 to 0');
+
+DELETE FROM `conditions` WHERE `SourceTypeOrReferenceId` = 22 AND `SourceEntry` = 27277 AND `SourceId` = 0;
+
+DELETE FROM `creature_text` WHERE `CreatureID` = 27277;
+INSERT INTO `creature_text` (`CreatureID`, `GroupID`, `ID`, `Text`, `Type`, `Language`, `Probability`, `Emote`, `Duration`, `Sound`, `BroadcastTextId`, `TextRange`, `comment`) VALUES
+(27277, 0, 0, 'Alright Ben. We\'ve retrieved the amberseeds again. You know the drill.', 12, 0, 100, 0, 0, 0, 26350, 0, 'Master Woodsman Anderhol // Master Woodsman Anderhol');
+
+DELETE FROM `creature_text` WHERE `CreatureID` = 27071;
+INSERT INTO `creature_text` (`CreatureID`, `GroupID`, `ID`, `Text`, `Type`, `Language`, `Probability`, `Emote`, `Duration`, `Sound`, `BroadcastTextId`, `TextRange`, `comment`) VALUES
+(27071, 0, 0, 'I know, I know. Back in the bucket....', 12, 0, 100, 0, 0, 0, 26351, 0, 'Benjamin Jacobs // Benjamin Jacobs');
+
+DELETE FROM `waypoints` WHERE (`entry` = 27071) AND (`pointid` = 1);
+DELETE FROM `waypoints` WHERE (`entry` = 27071) AND (`pointid` = 2);
+DELETE FROM `waypoints` WHERE (`entry` = 2707100) AND (`pointid` = 1);
+DELETE FROM `waypoints` WHERE (`entry` = 2707100) AND (`pointid` = 2);
+DELETE FROM `waypoints` WHERE (`entry` = 2707100) AND (`pointid` = 3);
+DELETE FROM `waypoints` WHERE (`entry` = 2707100) AND (`pointid` = 4);
+DELETE FROM `waypoints` WHERE (`entry` = 2707100) AND (`pointid` = 5);
+DELETE FROM `waypoints` WHERE (`entry` = 2707100) AND (`pointid` = 6);
+DELETE FROM `waypoints` WHERE (`entry` = 2707100) AND (`pointid` = 7);
+DELETE FROM `waypoints` WHERE (`entry` = 2707100) AND (`pointid` = 8);
+DELETE FROM `waypoints` WHERE (`entry` = 2707100) AND (`pointid` = 9);
+DELETE FROM `waypoints` WHERE (`entry` = 2707100) AND (`pointid` = 10);
+DELETE FROM `waypoints` WHERE (`entry` = 2707100) AND (`pointid` = 11);
+DELETE FROM `waypoints` WHERE (`entry` = 2707100) AND (`pointid` = 12);
+DELETE FROM `waypoints` WHERE (`entry` = 2707100) AND (`pointid` = 13);
```

Comment:
DELETE FROM `waypoints` WHERE (`entry` = 2707100) AND (`pointid` BETWEEN 1 AND 13);